CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.10.1
+
+A patch release, which fixes/improves some job metadata fields:
+
+- `dist_git_branch` (see `0.10.0`) has been renamed to `dist_git_branches` and it accepts also a list of values.
+- New `scratch` and `branch` keys.
```

Comment:
This line does not make sense for users without context. We can remove that line or what about something like this?

```suggestion
- New `scratch` option that will be used for koji builds.
- New `branch` option that will be used for specifying a branch for which we want to run builds.
```

Reply:
`mergeit` merged it, but I'll fix it in another commit

Reply:
No problem, that was not so important.